sys/net/ble/skald/skald.c

```diff
@@ -46,7 +56,7 @@
 #define ADV_AA                  (0x8e89bed6)    /* access address */
 #define ADV_CRC                 (0x00555555)    /* CRC initializer */
 
-static const uint8_t _adv_chan[] = SKALD_ADV_CHAN;
+static uint8_t _adv_chan[] = CONFIG_SKALD_ADV_CHAN;
```

Comment:
I think this may be messing with the definition of `ADV_CHAN_NUMOF`, which is used to traverse through the channels later.

Reply:
Thanks for the suggestion. Added fixup.